app/main.py

```diff
@@ -1,8 +1,47 @@
+from decimal import Decimal
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: float) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        cost = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                cost += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return cost
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if self.clean_power <= car.clean_mark:
+            return 0
+        price = round(float(
+            car.comfort_class * (self.clean_power - car.clean_mark)
+            * Decimal(str(self.average_rating))
```

Comment:
There is no need to use `Decimal` for this task, omit it.

Reply:
In case you meet edge cases, then you can leave it as is.

Reply:
not fixed

Reply:
the next example in the task
------------
bmw = Car(3, 3, 'BMW')
audi = Car(4, 9, 'Audi')
mercedes = Car(7, 1, 'Mercedes')

ws = CarWashStation(6, 8, 3.9, 11)

income = ws.serve_cars([
    bmw,
    audi,
    mercedes
])

income == 41.7
-------------
without using Decimal gives result of income as 41.6
cause
print((ws.calculate_washing_price(bmw)))  ->9.8
print((ws.calculate_washing_price(audi)))  ->
print((ws.calculate_washing_price(mercedes)))  ->31.8
(31.849999999999997868371792719699442386627197265625 rounded to 1 decimal)